web/src/LogPane.tsx

```diff
@@ -21,6 +21,15 @@ type LogPaneProps = {
   isSnapshot: boolean
 }
 
+type LogPaneState = {
+  // The number of log lines to display
+  renderWindow: number
+}
+
+const renderWindowDefault = 50
+const renderWindowStep = 50
+const blankLogLineHeight = 30
```

Comment:
Nit: We could use styled-components for the logLine CSS, and use this const value, to ensure it's accurate. For now, though, maybe a comment to note that it's just a value that tracks the computed height of each log line? (Which is currently based on font size and line height.)

Reply:
That wouldn't work, because a log line can have multiple lines of text, so the height isn't fixed

Reply:
added come comments for clarity